libs/rxidb/README.md

```diff
@@ -0,0 +1,220 @@
+<h1 align="center"> RxIDB ⚙️</h1>
+
+RxIDB is a library for storing and retrieving data from indexedDB databases, using an [RxJS](https://github.com/ReactiveX/rxjs) based API.
+
+It supports:
+
+✅ &nbsp;Creating new IndexedDB databases and Object Stores <br/>
+✅ &nbsp;Making CRUD operations on the above-mentioned stores <br/>
+✅ &nbsp;Read data out of indexedDB <br/>
```

Comment:
Maybe "Reading" instead of "Read"? To keep things consistent with Creating and Making.